README.md

```diff
@@ -80,14 +80,25 @@ The basic comparisons are:
 * `>=`: greater than or equal to
 * `<=`: less than or equal to
 
-_Note, according to the Semantic Version specification pre-releases may not be
-API compliant with their release counterpart. It says,_
+## Working With Pre-release Versions
 
-> _A pre-release version indicates that the version is unstable and might not satisfy the intended compatibility requirements as denoted by its associated normal version._
+According to the Semantic Version specification pre-releases may not be
```

Comment:
I initially arrived to the semver repo without a clear conception of what a pre-release was, perhaps you could help the former me by adding a small parenthesis clarification.

> According to the Semantic Version specification pre-releases (`-alpha`, `-beta` etc.) may not be API compliant with their release counterpart. It says,

Reply:
I agree -- a quick mention might help. I'd consider it a non-blocker, though. Otherwise, this looks great.